src/plugins/unified_search/public/search_bar/search_bar.test.tsx

```diff
@@ -6,35 +6,22 @@
  * Side Public License, v 1.
  */
 
-import React from 'react';
-import SearchBar from './search_bar';
+import React, { ReactNode } from 'react';
```

Comment:
Updated to React Testing Library.

Added more tests to more fully describe behavior.